packages/web-components/@groupby/elements-search-box/test/unit/search-box.test.ts

```diff
@@ -223,6 +255,16 @@ describe('SearchBox Component', () => {
     });
   });
 
+  describe('handleSearchClick', () => {
+    it('should trigger a search event to be omitted with an origin of search', () => {
```

Comment:
I'm not sure I understand this description... should this be 'emitted' not 'omitted'?